src/rules/no-if.js

```diff
@@ -0,0 +1,88 @@
+import { getDocsUrl, getNodeName, isTestCase, testCaseNames } from './util';
+
+const isTestArrowFunction = node =>
+  node !== undefined &&
+  node.type === 'ArrowFunctionExpression' &&
+  node.parent.type === 'CallExpression' &&
+  testCaseNames.has(getNodeName(node.parent.callee));
+
+export default {
+  meta: {
+    docs: {
+      description: 'Disallow conditional logic',
+      category: 'Best Practices',
+      recommended: false,
+      uri: getDocsUrl('jest/no-if'),
+    },
+    messages: {
+      noIf: [
+        'Tests should not contain if statements.',
```

Comment:
I wonder if the first sentence (for both messages) is enough. If people want more background, they can open up the docs

Reply:
@SimenB makes sense to me 👍